windows-installer/README.md

```diff
@@ -0,0 +1,89 @@
+Windows installer script
+========================
+
+Overview
+--------
+
+The windows installer script found in this directory is capable of building a
+windows installer executable capable of installing both 32 and 64 bit openssl
+binaries, along with their corresponding development headers
+
+Requirements
+------------
+
+* [NSIS](https://nsis.sourceforge.io/Main_Page) version 3.0.8 or later
+* Windows 2022 or later
+* The Windows SDK
+  - The makecert.exe utility (to demonstrate installer signing)
+  - The Pvk2Pfx.exe utility (to demonstrate installer signing)
+  - The SignTool.exe utility (to demonstrate installer signing)
+
+Notes on Signing
+----------------
+
+Installer signing is demonstrated here using self signed certificates. Do not
+use this signed code in a deployment as the generated certificate should not be
+trusted.  However, if you wish to observe this signed installer in operation,
+the generated certificate may be imported to the local trust store following the
+instructions
+[here](https://learn.microsoft.com/en-us/windows/win32/appxpkg/how-to-create-a-package-signing-certificate).
+at your own risk.
+
+Installer Build Prerequisites
+-----------------------------
+
+1) Build Openssl from the parent of this directory:
+    a) cd /path/to/openssl/source/root
+    b) mkdir \_build64
+    c) cd \_build64
+    d) perl ..\Configure [options] VC-WIN64A
+    e) nmake
+    f) repeat steps a-e substituting \_build32 for \_build64 to build VC-WIN32
+

```

Comment:
I think there should be step a) saying:
` a) clone/extract openssl sources^`
the following steps should be renamed accordingly. also (now) step b) should read like
`b) cd \path\to\your\openssl`
we are on windows so back-slash is separator. I also prefer the last path component to be openssl using `openssl\root` might confusing because there is no `root` directory in openssl sources.